### ISSUE TITLE
Can't get this to work on a Mac

### ISSUE THREAD

**Issue by ginko** [mihai-vlc/ChangedFilesTracker#1]
Hello, have you tested this plugin on a mac? I tried to add the path to the config file but it does not seem to work on Sublime 3

**Comment by mihai-vlc** [comment]
Hi,

I don't have a mac but this extension doesn't use anything os specific so it should work.

Maybe the path to the folder is not correct.
Open the console in sublime (ctrl/cmd + `) and type `view.file_name()` it should list the path to the current file.
Then compare that with what you currently have in the sublime settings file.

**Comment by ginko** [comment]
Ok so maybe I'm misunderstanding how this plugin works. Is it creating a
new folder in the directory called _changed and puts the changed files in
there?

On Wed, Aug 31, 2016 at 1:17 PM, Mihai Ionut Vilcu <notifications@github.com

**Comment by mihai-vlc** [comment]
yes, it basically copies the files from one place to another when you save them.

Just out of curiosity what behavior were you looking for ?

**Comment by ginko** [comment]
Ah Ok, I though it would highlight the folder and the file where the change
occurred. Thanks.
